src/test/java/org/zalando/nakadi/service/subscription/StreamingContextTest.java

```diff
@@ -29,8 +30,8 @@ public void onException(final Exception ex) {
             }
 
             @Override
-            public void streamData(final byte[] data) throws IOException {
-
+            public OutputStream getOutputStream() {
+                return null;
```

Comment:
> Just saying that there were no unit tests for the previous streamData method either. :) 

Fair enough! I can do some local testing on this branch and we can take it from there 😄